#request
int8 num_layer
int16 num_course
---
#response
geometry_msgs/PoseArray single_course_poses
visualization_msgs/MarkerArray single_course_marker